modules/ROOT/pages/migration-core-choice.adoc

```diff
@@ -5,14 +5,16 @@ endif::[]
 
 // sme: DF, author: sduke?
 
-Little changed between Choice router for Mule 3 and Mule 4 except for:
+The Choice router didn't change much from Mule runtime engine (Mule) 3 to Mule 4:
 
-* DataWeave is now the expression language so the when statements are actually going to use it instead of MEL
-* The `<otherwise>` block is now optional
+* DataWeave is now the expression language, so `<when>` attributes use it instead of MEL.
+* The `<otherwise>` block is now optional.
 
 == Expression Language for the When Block
 
-The when expressions need to be changed from MEL to DataWeave. This also mean that you can leverage DataWeave's ability to interpret any format to avoid unnecesary convertions. For example, suppose a flow in which a JSON document representing a user is posted through HTTP and you want to check if the person is underage:
+Change `<when>` expressions from MEL to DataWeave. As an added benefit, you can leverage DataWeave's ability to interpret any format to avoid unnecesary convertions.
+
+For example, suppose a flow in which a JSON document representing a user is posted through HTTP and you want to check if the person is underage.
```

Comment:
Does "underage" translate? Just curious.

Reply:
not sure so fixed it "is under a specified age limit (underage)." the example uses underage, that's why I added in parens.